templates/_helpers.yaml

```diff
@@ -78,12 +78,16 @@
   {{ end }}
 {{- end }}
 
+{{ define "airflow_host" -}}
+  {{ printf "deployments.%s" .Values.ingress.baseDomain }}
+{{- end }}
+
 {{ define "airflow_url" -}}
-{{ .Release.Name }}-airflow.{{ .Values.ingress.baseDomain }}
+  {{ .Release.Name }}-airflow.{{ .Values.ingress.baseDomain }}
```

Comment:
@andriisoldatenko can we un-indent the helpers in here? I think "standard" practice is no indentation on these. For example: https://github.com/helm/charts/blob/master/stable/prometheus/templates/_helpers.tpl

Reply:
@andriisoldatenko thinking more about these helper names, sorry!

What do you think about these? Just trying to come up with something that is consistent and makes sense.
`airflow_host` = `deployments_subdomain`
`airflow_url` = `airflow_subdomain`
`flower_url` to `flower_subdomain`

Reply:
@schnie ptal